test/utils/query.ts

```diff
@@ -51,7 +64,11 @@ describe('query utils', () => {
 					get: sandbox.stub().resolves(response),
 				},
 			};
-			queryResult = query(apiClient, defaultEndpoint, defaultParameters);
+			queryResult = query(
+				(apiClient as unknown) as APIClient,
```

Comment:
unknown needed here?

Reply:
it needed to convert to unknown first otherwise it complains...